lib/ssh_scan/api.rb

```diff
@@ -150,14 +152,7 @@ class API < Sinatra::Base
           return {"accepted" => "false"}.to_json
         end
 
-        settings.db.find_work_result(worker_id, uuid)
-      end
-
-      get '/__version__' do
-        {
-          :ssh_scan_version => SSHScan::VERSION,
-          :api_version => SSHScan::API_VERSION,
-        }.to_json
+        settings.db.add_scan(worker_id, uuid, JSON.parse(request.body.first).first)
```

Comment:
If you ever task a multi-ip scan to a worker, this means that you will only ever record the first result of potentially many results that would be posted back.  I think this is ok if the expectation is that you'll only ever task a single target at a time via the API.